### PR TITLE
Milestone 2: Works section About me section

### PR DESCRIPTION
**Pull request by AlexRS90**
- Added Works section
- Added AboutMe section
- Added new images for those sections
- Updated index.html file
- Updated style.css file
- Updated README.md

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -75,3 +75,360 @@
   margin-bottom: 120px;
```
Comment (carlos-ssh) [inline style.css:75]: [OPTIONAL]
Work professionally in your future projects and improve your CSS code using DRY method.

**Review by carlos-ssh — APPROVED** [review]
Hi @AlexRS90 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (AlexRS90) [reply style.css:75]: Thanks a lot, Carlos!
I'm happy to read you, and of course, I will use DRY method. 🤓 

Also, I'm from Xalapa, Veracruz, it's nice to see Mexicans 😃

File: style.css [hunk 2]
```diff
@@ -75,3 +75,360 @@
...
+
+.grid3 p {
+  font-family: 'Inter', sans-serif;
+  font-size: 16px;
+  font-weight: 400;
+  line-height: 24px;
+  margin-top: 12px;
+  margin-left: 16px;
+  margin-right: 16px;
+  margin-bottom: 0%;
```
Comment (micheaol) [inline style.css:150]: Just a suggestion, try to make your code DRY. It makes your code neat.

**Review by micheaol — COMMENTED** [review]
Good job,
Recommended ♻️  
Please find my comment below.

**Pull request merged**